client/src/partials/stock/entry/templates/quantity.tmpl.html

```diff
@@ -0,0 +1,3 @@
+<div class="ui-grid-cell-contents">
+  {{ row.entity.givenQuantity || row.entity.quantity }}
```

Comment:
The quantity can come from a purchase order as default quantity or it can be calculate by quantities of lots